x-pack/plugins/security_solution/public/cases/pages/configure_cases.tsx

```diff
@@ -61,7 +61,7 @@ const ConfigureCasesPageComponent: React.FC = () => {
         <WhitePageWrapper>
           {cases.getConfigureCases({
             userCanCrud: userPermissions?.crud ?? false,
-            owner: [APP_ID],
+            owner: [APP_UI_ID],
```

Comment:
Ok I think you get the idea 😆 just search for `owner: ` and those need to be switched back to `owner: [APP_ID]`